### PR TITLE
Tob screenshots

### PR DESCRIPTION
**Pull request by MagicfTail**
Add support for taking screenshots of theatre of blood rewards

Relies on #4245

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java [hunk 1]
```diff
@@ -373,6 +387,17 @@ public void loadWidgets(WidgetLoaded event)
...
 				break;
 			}
+			case THEATRE_OF_BLOOD_REWARD_GROUP_ID:
+			{
+				if (theatreOfBloodNumber == null)
+				{
+					return;
+				}
+
+				fileName = "Theatres of Blood(" + theatreOfBloodNumber + ")";
```
[the remark this answers is not in the document]
Comment (MagicfTail) [reply runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java]: ? That's what it's called http://oldschoolrunescape.wikia.com/wiki/Theatre_of_Blood

[the remark this answers is not in the document]
Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java]: Yes, but you have `Theatres` there

[the remark this answers is not in the document]
Comment (MagicfTail) [reply runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java]: Both raids and chambers of Xeric were plural, and it sounds to me like it should be 🤷‍♀️

[the remark this answers is not in the document]
Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java]: But chambers of xeric are plural by default.

[the remark this answers is not in the document]
Comment (MagicfTail) [reply runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java]: I guess I'll change it, rip all the screenshots I already have :(

**Pull request merged**